libraries/math/src/approximations.rs

```diff
@@ -41,6 +41,31 @@ pub fn sqrt<T: PrimInt + CheckedShl + CheckedShr>(radicand: T) -> Option<T> {
     Some(result)
 }
 
+/// Calculate the normal cdf of the given number
+///
+/// Code lovingly adapted from the excellent work at:
+///
+/// <https://www.hrpub.org/download/20140305/MS7-13401470.pdf>
+///
+/// The algorithm is based on the implementation in the paper above.
```

Comment:
In numerical computing, it's crucial to provide some guarantees about how accurate the approximation is, ie. out to 3 digits according to the paper.